src/opustrainer/trainer.py

```diff
@@ -593,12 +593,13 @@ def state(self) -> EpochTrackerState:
 
 Out = TypeVar('Out')
 
-def trace_map(fn: Callable[[In], Out], items: Iterable[In]) -> Iterable[Out]:
-    for n, item in enumerate(items):
+def try_trace_map(fn: Callable[[In], Out], items: Iterable[In]) -> Iterable[Out]:
+    for item in items:
         try:
             yield fn(item)
         except Exception as exc:
-            raise Exception(f'Exception while processing item {n}: {item!r}') from exc
+            logger.log(f'Exception while processing line, skipping: {item!r}', 'WARNING',
```

Comment:
This should ideally be `log_once` otherwise we would get spammed everytime we loop through the dataset. (I assume the exception kwards would be the same always)